packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -1061,15 +1069,20 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
     );
 
     if (!widget.constrained) {
+      child = OverflowBox(
```

Comment:
There is definitely a use case for needing a ClipRect with `constrained` set to false.  If you want the InteractiveViewer to be small (like a small window) but the child to be big and pannable:

<img width="229" alt="Screen Shot 2021-01-07 at 9 46 50 AM" src="https://user-images.githubusercontent.com/389558/103926270-8a13c000-50cd-11eb-9581-8ddc613675b4.png">

Then you want  a ClipRect and `constrained` false.  Can we use a ClipRect with the clipBehavior parameter in both cases?

Reply:
Absolutely ! Actually I did this PR to have both and control where my ClipRect is